packages/runtime-core/src/component.ts

```diff
@@ -224,6 +224,10 @@ export interface ComponentInternalInstance {
   parent: ComponentInternalInstance | null
   root: ComponentInternalInstance
   appContext: AppContext
+  /**
+   * The DOM element that app is mounted to, or shadowRoot of custom element.
+   */
+  container: any
```

Comment:
```suggestion
  container: HTMLElement | null
```

Reply:
Given only setting for CE, the type can be narrowed.